packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.cnstyles.ts

```diff
@@ -187,51 +187,3 @@ export const getMenuItemStyles = memoizeFunction(
     return concatStyleSets(menuItemStyles);
   }
 );
-
-export const getStyles = memoizeFunction(
```

Comment:
Removed this style function as it's never used in our repo. @KevinTCoughlin can you confirm that this is not the issue you had before?

Reply:
This isn't the _private_ API that broke one of our partners last week, however if it does not impact your PR I would advise keeping it in this PR. We can then remove in a separate PR and verify separately.

Reply:
As far as I can tell this is unused across OUIFR. Likely leftover from when we were refactoring components piece-meal #5356.